community/sway/etc/sway/config.d/92-enable-mako-theme.conf

```
@@ -1,5 +1,4 @@
 exec_always {
   '[ ! -f "$HOME/.config/mako/config" ] && mkdir -p $HOME/.config/mako && cat /usr/share/sway/themes/templates/mako > $HOME/.config/mako/config'
-  'sed -i -e "/border-color/c border-color=ACCENT_COLOR" -e "/text-color/c text-color=TEXT_COLOR" -e "/font/c font=TERM_FONT" -e "/background-color/c background-color=BACKGROUND_COLOR" $HOME/.config/mako/config'
-  'sed -i -e $theme_template_script $HOME/.config/mako/config'
+  'sed -i -e "/border-color/c border-color=ACCENT_COLOR" -e "/text-color/c text-color=TEXT_COLOR" -e "/font/c font=TERM_FONT" -e "/background-color/c background-color=BACKGROUND_COLOR" $HOME/.config/mako/config && sed -i -e $theme_template_script $HOME/.config/mako/config'
```

supposed what you did works, shouldn't this be fine too then?
```suggestion
  'sed -i -e "/border-color/c border-color=ACCENT_COLOR" -e "/text-color/c text-color=TEXT_COLOR" -e "/font/c font=TERM_FONT" -e "/background-color/c background-color=BACKGROUND_COLOR" -e $theme_template_script $HOME/.config/mako/config'
```

---

(I am on holidays without sway at hand - perhaps @Mathew-D and @simon-bueler need to try this out...

---

That doesn't not work as sed will only go through the file once that way and will not replace the same line twice. I was going to use the vars from the theme template files directly instead of using the $them_tempate_script but the background color use two different vars for the color. 
blue:
set $theme_template_script "s/GTK_THEME/$gtk-theme/g; s/ICON_THEME/$icon-theme/g; s/CURSOR_THEME/$cursor-theme/g; s/GUI_FONT/$gui-font/g; s/KVANTUM_THEME/$kvantum-theme/g; s/BACKGROUND_COLOR/$base00/g; s/ACCENT_COLOR/$base0D/g; s/TEXT_COLOR/$base06/g; s/TERM_FONT/$term-font/g;"% 

green
set $theme_template_script "s/GTK_THEME/$gtk-theme/g; s/ICON_THEME/$icon-theme/g; s/CURSOR_THEME/$cursor-theme/g; s/GUI_FONT/$gui-font/g; s/KVANTUM_THEME/$kvantum-theme/g; s/BACKGROUND_COLOR/$base00/g; s/ACCENT_COLOR/$base0C/g; s/TEXT_COLOR/$base06/g; s/TERM_FONT/$term-font/g;"% 

If the ACCENT_COLOR used the same var we could have just replaced once with the vars from the theme definitions.

---

Just to let you know I am currently using the version I posted on this pull with them one line and rebooted about 20 times to double check and haven't had any problems.

---

ok, then lets go with it. forgot that we kinda would need to do that recursively.